portal/server/helpers/isChecker.helper.js

```diff
@@ -1,6 +1,8 @@
 const { CHECKER } = require('../constants/roles');
 const { equalArrays } = require('./equalArrays.helper');
 
+// TODO DTFS2-6635: this function returns false if a user has checker plus another role,
+// which is surprising. Is this correct?
```

Comment:
highlighting TODO

Reply:
Resolved this to be investigated separately since it's out of scope for this ticket